src/types.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * PWA assets generation and injection options.
+ */
+export interface PWAAssets {
+  /**
+   * PWA assets generation and injection.
+   *
+   * If `true` the plugin will search for the pwa assets generator configuration file in the root directory of your project:
+   * - pwa-assets.config.js
+   * - pwa-assets.config.mjs
+   * - pwa-assets.config.cjs
+   * - pwa-assets.config.ts
+   * - pwa-assets.config.cts
+   * - pwa-assets.config.mts
+   *
+   * If using a string path, it should be relative to the root directory of your project.
+   *
+   * @default true
+   * @see https://vite-pwa-org.netlify.app/assets-generator/cli.html#configurations
+   */
+  path?: true | string;
+  /**
+   * Should the plugin override the PWA web manifest icons' entry?
+   *
+   * Ff you provide `true`, this plugin will use `public/manifest.webmanifest`.
+   *
+   * If you provide a string, it will be used as the path to your web manifest file: it must be a relative path inside the public folder.
+   *
+   * @default false
+   */
+  overrideManifestIcons?: boolean;
```

Comment:
include hint about modifying the manifest source file, we cannot modify in the output dirrectory since Vite will copy public folder after build finish and before cli process finish (manifest will be overriden)

Reply:
maybe we should add web manifest options here: just a copy/paste from Vite PWA types: json file is a pain, with a custom manifest entry in the plugin options the user can provide the manifest in js/ts or via custom json file

Reply:
I mean I did this and it was working! I modified the manifest file and there was no problem as far as I know! 

We cannot do types though, because the basic qwik examples all have them and  I don't want so much modification for the user, everything should work out of the box.

Reply:
you're forcing to override the icons: what happens if the user provides the pwa icons and the icons entry in the web manifest?